src/formatDate.js

```diff
@@ -50,7 +50,27 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateFormat = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
`fromFormat[fromFormat.length - 1] ` is usually recommended to replace into separate varriable) 
For example, `const oldSeparator =  fromFormat[fromFormat.length - 1] `